### DISCUSSION TITLE
对于单个组件，你还有什么意见或建议？

### DISCUSSION THREAD

**Discussion by LeeJim** [Tencent/tdesign-miniprogram#845] [Show and tell]
主要为了记录一些组件当前存在的问题，但一时半会可能没法解决的问题。
先记录一下，避免遗漏了。

**Comment by LeeJim** [comment]
## TabBar

支持 subTab，但 Tab Item 目前只有 active 状态，没有 spread(展开) 状态

**Comment by LeeJim** [comment]
pull-down-refresh 的动画操作改用 wxs 实现

**Reply by LeeJim** [reply]
还有 picker

**Comment by arbureva** [comment]
建议多出一些卡片组件
